packages/streamline/src/theme/appearance.ts

```diff
@@ -3,4 +3,7 @@ export type Appearance =
   | 'secondary'
   | 'danger'
   | 'neutral'
-  | 'disabled';
+  | 'disabled'
+  | 'placeholder'
```

Comment:
```suggestion
```
can you replace this with a new prop isSkeleton?
we want to avoid using appearance for this as it will be cleaner to make

```
<Tag appearance="secondary" isSkeleton />
```

Reply:
Done, but just wanted to note that it was not used for Tag but for Button. Normally should be fine, but it's worth updating again both iOS and Android screenshots (simu looks good)